translations/monero.ts

```diff
@@ -3701,7 +3701,7 @@ daemon to work correctly.</source>
     <name>tools::wallet2</name>
     <message>
         <location filename="../src/wallet/wallet2.cpp" line="113"/>
-        <source>Use daemon instance at &lt;host&gt;:&lt;port&gt;</source>
+        <source>Use daemon instance at &lt;host&gt; on default RPC port.</source>
```

Comment:
This seems wrong. The flag does expect the `<host>:<port>` format.

Reply:
You're right. 

I think my previous issue was due to using an older aeon-wallet-cli that was in one of the first releases of the aeon-gui.

Using latest releases fixes the double port issue I had previously that got me to this edit. Edited back to original language.